From 34c213ce7e7ef7e1f8fd91b686da655df43efca2 Mon Sep 17 00:00:00 2001
From: Nagaraju Mekala <nmekala@xilinx.com>
Date: Tue, 20 Apr 2021 21:22:06 +0530
Subject: [PATCH 30/38] [Patch, microblaze]:Changing the long to long long as
 in Windows long is 32-bit but we need the variable to be 64-bit

Signed-off-by :Nagaraju Mekala <nmekala@xilix.com>
---
 gas/config/tc-microblaze.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: git/gas/config/tc-microblaze.c
===================================================================
--- git.orig/gas/config/tc-microblaze.c
+++ git/gas/config/tc-microblaze.c
@@ -1017,7 +1017,7 @@ md_assemble (char * str)
   unsigned reg2;
   unsigned reg3;
   unsigned isize;
-  unsigned long immed = 0, immed2 = 0, temp;
+  unsigned long long immed = 0, immed2 = 0, temp;
   expressionS exp, exp1;
   char name[20];
   long immedl;
